manifests/charts/global.yaml

```diff
@@ -410,17 +296,6 @@ global:
   # value.
   meshID: ""
 
-  # Set the default behavior of the sidecar for handling outbound traffic from the application:
-  # ALLOW_ANY - outbound traffic to unknown destinations will be allowed, in case there are no
-  #   services or ServiceEntries for the destination port
-  # REGISTRY_ONLY - restrict outbound traffic to services defined in the service registry as well
-  #   as those defined through ServiceEntries
-  # ALLOW_ANY is the default in 1.1.  This means each pod will be able to make outbound requests
-  # to services outside of the mesh without any ServiceEntry.
-  # REGISTRY_ONLY was the default in 1.0.  If this behavior is desired, set the value below to REGISTRY_ONLY.
-  outboundTrafficPolicy:
-    mode: ALLOW_ANY
```

Comment:
what is the equivalent for this in 1.6 to configure outbound traffic policy?

Reply:
It's configured by mesh config directly, as in `--set meshConfig.outboundTrafficPolicy`